docs/DeveloperGuide.md

```diff
@@ -299,12 +299,19 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
```

Comment:
Do glossary terms need to be italicized? May need to standardize.

Reply:
Glossary terms shall remain unformatted. Edits will be made.